requirements/main.txt

```diff
@@ -2,3 +2,8 @@ numpy>=1.15.0
 cma>=2.6.0
 bayesian-optimization>=1.1.0
 typing_extensions>=3.6.6
+torch
+gpytorch
+pygame
+gym
+matplotlib
```

Comment:
those requirements should *not* be included by the default install, they are definitely too complex for just one case (and probably most of them should actually be in bench.txt)

Please notify me if you want to move forward with this PR